pandas/tests/io/formats/test_to_html.py

```diff
@@ -78,6 +78,36 @@ def test_to_html_with_col_space(col_space):
         assert str(col_space) in h
 
 
+def test_to_html_with_column_specific_col_space():
+    df = DataFrame(np.random.random(size=(3, 3)), columns=["a", "b", "c"])
+
+    msg = (
```

Comment:
Same comment here - if could split the error message testing into another test would be ideal